### DISCUSSION TITLE
tensorrt image with coral?

### DISCUSSION THREAD

**Discussion by slashgob** [blakeblackshear/frigate#13285] [General]
I just realised i'm using the ghcr.io/blakeblackshear/frigate:stable-tensorrt image in Unraid but using a usb coral for detection. Do I need the tensorrt image?

My config:
detectors:
  coral:
    type: edgetpu
    device: usb

is it possible to use both Coral and Nvidia GPU for detection? Is there any advantage in doing this?

**Comment by NickM-27** [comment]
No you don't need it, no mixing detector types is not supported
